src/main/resources/generate_from_proto.sh

```diff
@@ -4,8 +4,23 @@
 
 set -e
 
-python3 -m grpc_tools.protoc -I../proto \
---python_out=../python/pb --grpc_python_out=../python/pb \
-../proto/service.proto \
-../proto/github.com/gogo/protobuf/gogoproto/gogo.proto \
-../proto/gopkg.in/bblfsh/sdk.v1/uast/generated.proto
+# Python server
+
+pip3 install grpcio==1.10.0 grpcio-tools==1.10.0
+python3 -m grpc_tools.protoc -Isrc/main/proto \
+--python_out=src/main/python/pb --grpc_python_out=src/main/python/pb \
+src/main/proto/service.proto \
+src/main/proto/github.com/gogo/protobuf/gogoproto/gogo.proto \
+src/main/proto/gopkg.in/bblfsh/sdk.v1/uast/generated.proto
+
+# Scala client
+
+wget https://github.com/scalapb/ScalaPB/releases/download/v0.7.1/scalapbc-0.7.1.zip
+unzip scalapbc-0.7.1.zip
+scalapbc-0.7.1/bin/scalapbc -Isrc/main/proto \
```

Comment:
It's very nice to have self-contained, end to end example, but in case user already have `scalapbc`available on there env, do you think we could download/cleanup it conditional? 

Here is example how we handle some conditions in with [spark-submit](https://github.com/src-d/gemini/blob/master/hash#L23).

Reply:
I don't think it's a good idea. We need exact version of scalapbc (another version can potentially generate different code). I can write complex logic with checking for scalapbc, then version, then download if needed and so on. But because this script would be run very very rare I don't see advantages to make it so complex to save few kilobytes of traffic for a developer

Reply:
Good point, checking version in .sh does not make sense indeed. 

Let's then at least check if the zip file is already present in FS - the reason is that having an option for this automation to work without network access is useful and a good engineering practice.